docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -154,201 +154,201 @@ input_dim: 3
+#### 34. What does the message "Unsupported match kind.... Match kinds "points" or "scope" are supported only" mean? <a name="question-34"></a>
 
-While using configuration file to implement a TensorFlow\* front replacement extension, an incorrect match kind was used. Only `points` or `scope` match kinds are supported. Please refer to [Model Optimizer Extensibility](customize_model_optimizer/Customize_Model_Optimizer.md) for more details.
+While using configuration file to implement a TensorFlow front replacement extension, an incorrect match kind was used. Only `points` or `scope` match kinds are supported.  For more details, refer to the [Model Optimizer Extensibility](customize_model_optimizer/Customize_Model_Optimizer.md) guide.
 
-#### 36. What does the message "Cannot write an event file for the TensorBoard to directory" mean? <a name="question-36"></a>
+#### 35. What does the message "Cannot write an event file for the TensorBoard to directory" mean? <a name="question-35"></a>
 
-Model Optimizer tried to write an event file in the specified directory but failed to do that. That could happen because the specified directory does not exist or you do not have enough permissions to write in it.
+Model Optimizer tried to write an event file in the specified directory but failed to do that. That could happen because the specified directory does not exist or you do not have permissions to write in it.
 
-#### 37. What does the message "There is no registered 'infer' function for node  with op = .. . Please implement this function in the extensions" mean? <a name="question-37"></a>
+#### 36. What does the message "There is no registered 'infer' function for node  with op = .. . Please implement this function in the extensions" mean? <a name="question-36"></a>
 
-Most likely, you tried to extend Model Optimizer with a new primitive, but did not specify an infer function. For more information on extensions, see [OpenVINO™ Extensibility Mechanism](../../Extensibility_UG/Intro.md).
+Most likely, you tried to extend Model Optimizer with a new primitive, but you did not specify an infer function. For more information on extensions, see the [OpenVINO&trade; Extensibility Mechanism](../../Extensibility_UG/Intro.md) guide.
 
-#### 38. What does the message "Stopped shape/value propagation at node" mean? <a name="question-38"></a>
+#### 37. What does the message "Stopped shape/value propagation at node" mean? <a name="question-37"></a>
 
-Model Optimizer cannot infer shapes or values for the specified node. It can happen because of a bug in the custom shape infer function, because the node inputs have incorrect values/shapes, or because the input shapes are incorrect.
+Model Optimizer cannot infer shapes or values for the specified node. It can happen because of a bug in the custom shape infer function or the node inputs have incorrect values/shapes, or the input shapes are incorrect.
```

Comment:
```suggestion
Model Optimizer cannot infer shapes or values for the specified node. It can happen because a bug exists in the custom shape infer function, the node inputs have incorrect values/shapes, or the input shapes are incorrect.
```